doorway-ui-components/src/overlays/Modal.scss

```diff
@@ -109,6 +109,10 @@
   gap: var(--bloom-s5);
   justify-content: var(--footer-justify);
   align-items: center;
+
+  @media (max-width: $screen-md) {
+    margin: 0 0 3rem 0;
```

Comment:
Can you just add margin-bottom since there's just one, to make it easier to understand what margin that is, and us var(--bloom-s12)?